base/reduce.jl

```diff
@@ -48,10 +48,8 @@ function _foldl_impl(op::OP, init, itr) where {OP}
     y = iterate(itr)
```

Comment:
In theory this change is essentially identical except that this now unrolls two iterations instead of one, so the comment now needs to be adjusted, since it states the loop is only unrolled once

Reply:
I've added an additional comment before the loop